# awesome\-mave


List of software packages (and the people developing these methods) for multiplexed assays of variant effects.
[Contributions welcome](https://github.com/Constantiam-Biosciences-Inc/awesome-mave/blob/master/CONTRIBUTING.md)...


## Contents



* [awesome\-mave](#awesome-mave)
	+ [Contents](#contents)
	+ [Software Packages](#software-packages)
	+ [In Silico Tools](#in-silico-tools)
	+ [Analysis Tools](#analysis-tools)
	+ [Applications](#applications)
	+ [Websites / Portals](#websites-portals)
	+ [Databases](#databases)



## Software Packages


## In Silico Tools


* [REVEL](https://sites.google.com/site/revelgenomics/) \- \[.] \- An ensemble method for predicting the pathogenicity of missense variants based on a combination of scores from 13 individual tools
* [AlphaMissense](https://alphamissense.hegelab.org/) \- \[.] \- A deep learning model that builds on the protein structure prediction tool AlphaFold2
* [CADD](https://cadd.gs.washington.edu/) \- \[.] \- CADD is a tool for scoring the deleteriousness of single nucleotide variants, multi\-nucleotide substitutions as well as insertion/deletions variants in the human genome.
* [GV\-Rep](https://github.com/bowang-lab/genomic-FM) \- \[.] \- A Large\-Scale Dataset for Genetic Variant Representation Learning


## Analysis Tools


* [Enrich2](https://enrich2.readthedocs.io/en/latest/) \- \[.] \- Enrich2 is a general software tool for processing, analyzing, and visualizing data from deep mutational scanning experiments.
* [mavetools](https://www.mavedb.org/docs/mavetools/index.html) \- \[.] \- MaveTools is a pure Python Library for bioinformatics and computational biology. It features useful tools that can be applied to MAVE data


## Applications


## Websites / Portals


* [MAVEvidence](https://mavevidence.com/)  \- \[.] \- MAVEvidence is the premier functional evidence resource for the clinical genetics community.
* [MAVEQuest](https://mavequest.varianteffect.org/) \- \[.] \- MaveQuest: a web resource for planning experimental tests of human variant effects
* [cBioPortal](https://www.cbioportal.org/) \- \[.] \- The cBioPortal for Cancer Genomics is an open\-access, open\-source resource for interactive exploration of multidimensional cancer genomics data sets.
* [OncoKB](https://www.oncokb.org/) \- \[.] \- MSK's Precision Oncology Knowledge Base


## Databases


* [MAVERegistry](https://registry.varianteffect.org/)  \- \[.] \- MaveRegistry is a collaborative resource for sharing progress on Multiplexed Assays of Variant Effect (MAVE)
* [MaveDB](https://www.mavedb.org/) \- \[.] \- MaveDB is a public repository for datasets from Multiplexed Assays of Variant Effect (MAVEs), such as those generated by deep mutational scanning (DMS) or massively parallel reporter assay (MPRA) experiments
* [Protein Gym](https://proteingym.org/) \- \[.] \- ProteinGym is a collection of benchmarks aiming at comparing the ability of models to predict the effects of protein mutations
