# awesome\-mave


List of software packages (and the people developing these methods) for multiplexed assays of variant effects.
[Contributions welcome](https://github.com/Constantiam-Biosciences-Inc/awesome-mave/blob/master/CONTRIBUTING.md)...


## Contents



* [awesome\-mave](#awesome-mave)
	+ [Contents](#contents)
	+ [Software Packages](#software-packages)
	+ [In Silico Tools](#in-silico-tools)
	+ [Analysis Tools](#analysis-tools)



## Software Packages


## In Silico Tools


* [REVEL](https://sites.google.com/site/revelgenomics/) \- \[.] \- An ensemble method for predicting the pathogenicity of missense variants based on a combination of scores from 13 individual tools
* [AlphaMissense](https://alphamissense.hegelab.org/) \- \[.] \- A deep learning model that builds on the protein structure prediction tool AlphaFold2


## Analysis Tools


* [Enrich2](https://enrich2.readthedocs.io/en/latest/) \- \[.] \- Enrich2 is a general software tool for processing, analyzing, and visualizing data from deep mutational scanning experiments.
* [mavetools](https://www.mavedb.org/docs/mavetools/index.html) \- \[.] \- MaveTools is a pure Python Library for bioinformatics and computational biology. It features useful tools that can be applied to MAVE data
